package.json

```diff
@@ -7,7 +7,7 @@
     "start": "webpack -w & nodemon server",
     "start-server": "nodemon server",
     "build-client": "webpack",
-    "test": "mocha --compilers js:babel-register --watch-extensions js,jsx server/**/*.test.js"
```

Comment:
Annoyingly, excluding directories in a UNIX-y shell glob doesn't seem to be straightforward. This is by the way yet another reason why one often puts the _source code_ for a repo into a `src` folder. You don't have to do that, but if you at least have a couple specific folders that contain JS, you can opt in to those (instead of opting out of `node_modules` specifically).